### PR TITLE
Discovery: Add footer to the channels page

### PR DESCRIPTION
**Pull request by danigm**
This patch improves a bit the channels page with a header and the footer.

The header has the search button, but that button doesn't do anything yet, the functionality is not implemented here.

https://phabricator.endlessm.com/T32021

### CODE REVIEW

File: kolibri_explore_plugin/assets/src/views/ChannelsPage.vue [hunk 1]
```diff
@@ -114,4 +133,13 @@
     background: linear-gradient(to bottom, rgba($black, 0.075) 0, rgba($black, 0) 5px) #dee2e6;
   }
 
+  .header {
+    background: #dee2e6;
+    border-bottom: 1px solid #cccccc;
```
Comment (manuq) [inline kolibri_explore_plugin/assets/src/views/ChannelsPage.vue:138]: Please use colors from the SCSS theme.

File: packages/eos-components/src/assets/EndlessLogo.svg [hunk 2]
```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!-- Generator: Adobe Illustrator 19.2.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
```
Comment (manuq) [inline packages/eos-components/src/assets/EndlessLogo.svg:2]: Is this the same SVG asset that we removed in 9c075ba0? Please explain in the commit message that this file is added, and where is it coming from.

File: kolibri_explore_plugin/assets/src/views/ChannelsPage.vue [hunk 3]
```diff
@@ -28,6 +76,7 @@
   import commonCoreStrings from 'kolibri.coreVue.mixins.commonCoreStrings';
   import { ChannelCardGroup } from 'eos-components';
   import _ from 'underscore';
+  import EndlessLogo from 'eos-components/src/assets/EndlessLogo.svg';
```
[the remark this answers is not in the document]
Comment (danigm) [reply kolibri_explore_plugin/assets/src/views/ChannelsPage.vue:79]: I got the Endless logo svg from here:
https://github.com/endlessm/gnome-initial-setup/blob/master/gnome-initial-setup/pages/language/EndlessLogo.svg

And modified the file to remove the text.

File: packages/eos-components/src/assets/EndlessLogo.svg (hunk 2 above)
Comment (danigm) [reply packages/eos-components/src/assets/EndlessLogo.svg:2]: I got the Endless logo svg from here:
https://github.com/endlessm/gnome-initial-setup/blob/master/gnome-initial-setup/pages/language/EndlessLogo.svg

And modified the file to remove the text.

File: kolibri_explore_plugin/assets/src/views/ChannelsPage.vue (hunk 1 above)
Comment (danigm) [reply kolibri_explore_plugin/assets/src/views/ChannelsPage.vue]: ups, I got that values from figma, but I didn't notice that we've that values as bootstrap variables.

Comment (manuq) [reply kolibri_explore_plugin/assets/src/views/ChannelsPage.vue]: No problem! Great, we should always use the theme so we can change everything in the same single place. If we had a CI check for hardcoded colors I would already enabled it hehe.

File: packages/eos-components/src/assets/EndlessLogo.svg (hunk 2 above)
Comment (manuq) [reply packages/eos-components/src/assets/EndlessLogo.svg:2]: OK, next time let's ask assets to Joana.

**Pull request merged**